public/css/main.css

```diff
@@ -614,21 +614,41 @@ Button
 	width: 100%;
 }
 
-.btn-username {
+.username-btn {
 	color: #000000;
 	font-size: 18px;
 	font-weight: 600;
 }
 
 /*
 ======================================
-Errors
+Profile
 ======================================
 */
 
-.error-auth {
```

Comment:
Looks like you've deleted the error-auth css attribute thing, it's still in use. Add this back in. Test by entering wrong info [here](https://mora5651.squireproject.com/register)